Lib/profiling/sampling/sample.py

```diff
@@ -826,6 +940,13 @@ def main():
         dest="format",
         help="Generate Gecko format for Firefox Profiler",
     )
+    output_format.add_argument(
+        "--live",
+        action="store_const",
+        const="live",
+        dest="format",
+        help="Display live top-like statistics using ncurses",
```

Comment:
Not sure if ncurses is that commonly known. How about something like this:

```suggestion
        help="Display live top-like live statistics in a terminal UI",
```